File: sdks/java/basys.sdk/src/main/java/org/eclipse/basyx/aas/backend/connected/ConnectedVABModelMap.java
package org.eclipse.basyx.aas.backend.connected;

import org.eclipse.basyx.vab.core.proxy.VABElementProxy;
/**
 * "Connected" implementation of VABModelMap<V extends Object>
 * @author rajashek
 *
 */
public class ConnectedVABModelMap<V extends Object> extends ConnectedElement {

	public ConnectedVABModelMap(VABElementProxy proxy) {
		super(proxy);
	}
}

File: sdks/c++/basys.sdk.cc/src/aas/backend/connected/aas/ConnectedVABModelMap.cpp
/*
 * ConnectedVABModelMap.cpp
 *
 *      Author: wendel
 */

#include "ConnectedVABModelMap.h"

namespace basyx {
namespace aas {
namespace backend {

ConnectedVABModelMap::ConnectedVABModelMap(std::shared_ptr<vab::core::proxy::IVABElementProxy> proxy) :
  ConnectedElement(proxy)
{}

}
}
}